.github/workflows/release.yml

```diff
@@ -38,20 +39,18 @@ jobs:
           then
             changes=$(xmlstarlet fo -R "$(find . -name addon.xml.in)" | xmlstarlet sel -t -v 'string(/addon/extension/news)' | awk -v RS= 'NR==1')
           fi
-          changes="${changes//'%'/'%25'}"
-          changes="${changes//$'\n'/'%0A'}"
-          changes="${changes//$'\r'/'%0D'}"
-          changes="${changes//$'\\n'/'%0A'}"
-          changes="${changes//$'\\r'/'%0D'}"
-          echo "changes=$changes" >> $GITHUB_OUTPUT
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          echo "changes<<${EOF}" >> $GITHUB_OUTPUT
+          echo "$changes" >> $GITHUB_OUTPUT
+          echo "${EOF}" >> $GITHUB_OUTPUT
```

Comment:
i have already seen your proposed changes on other repos
but the brackets added to EOF var `${EOF}`
seem to be not required, so should appears as `$EOF`

Reply:
Yeah there is no grouping or expansion, so `$EOF` will work fine

Reply:
Ok, I removed the brackets. So we should be good now.